### DISCUSSION TITLE
Filesystem to Gitsync Migration

### DISCUSSION THREAD

**Discussion by antwacky** [apache/airflow#41223] [General]
### What do you see as an issue?

I am going to migrate airflow to use gitsync, where currently DAGs are baked into a custom container image.

I have a question around this, having spotted in the database that DAGs have a file path defined.

Is this a straightforward switch, is any more steps required after switiching the gitsync and will the existing DAGs be synchronized with those from the gitsync repo going forward?

Thanks

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by potiuk** [comment]
I have no idea how to help you, but just wanted you to know that using issues for that kind of requests is a bad idea - use discussions instead. I converted it to discussion for you - hopefully someone can help you.
